terraform/tutorial.md

```diff
@@ -0,0 +1,351 @@
+
+The Git repository you cloned has all the source code, so you can explore the implementation details of the application. See the repository [README](https://github.com/GoogleCloudPlatform/stackdriver-sandbox) for a guided tour.
+
+### Prerequisites
+
+You must have an active Google Cloud Platform Billing Account. If you already have one, you can skip this section.
+
+Otherwise, to create a GCP Billing Account, do the following:
+
+1. Go to the Google Cloud Platform [Console](https://console.cloud.google.com/) and sign in (if you have an account), or sign up (if you don't have an account).
+1. Select **Billing** from the navigation panel and follow the instructions.
+
+For more information, see ["Create a new billing account"](https://cloud.google.com/billing/docs/how-to/manage-billing-account).
+
+## Set up
+
+The installation process takes a few minutes. When it completes, you see a message like the following:
+
+```bash
+********************************************************************************
+Cloud Operations Sandbox deployed successfully!
+
+     Google Cloud Console GKE Dashboard: https://console.cloud.google.com/kubernetes/workload?project=<project ID>
+     Google Cloud Console Monitoring Workspace: https://console.cloud.google.com/monitoring?project=<project ID>
+     Hipstershop web app address: http://XX.XX.XX.XX
+     Load generator web interface: http://XX.XX.XX.XX
+```
+
+The URLs in this message tell you where to find the results of the installation.
+
+If a message does **not** appear, and the installation script is not able to run, then try running `./install.sh` in your Cloud Shell terminal once you have set-up your billing account.
+
+## Explore the Sandbox!
+
+In this tutorial, walk through a guided tour of products in Cloud Operations and explore how they can be used to work with an application.
+
+A User Guide with visuals can be used to follow along as well. The User Guide can be found [here](https://github.com/GoogleCloudPlatform/stackdriver-sandbox/blob/master/docs/README.md).
+
+Let's get started!
+
+## Explore your project in GCP
+
+In another browser tab, navigate to the GCP GKE Dashboard URL, which takes you to the Kubernetes Engine ([documentation](https://cloud.google.com/kubernetes-engine/docs/)) **Workloads** page for the project created by the installer.
+
+## Explore Cloud Monitoring
+
+Navigate to the GCP Monitoring Workspace URL, which takes you to the Cloud Monitoring ([documentation](https://cloud.google.com/monitoring)) **Workspace** page for your new project. The console may take some time to create a new workspace. Afterward, you'll be able to see a few dashboards generated through Cloud Operations tools.
```

Comment:
You can probably link to https://console.cloud.google.com/monitoring for workspace too